pkgs/top-level/all-packages.nix

```diff
@@ -11947,12 +11947,17 @@ let
   mendeley = callPackage ../applications/office/mendeley { };
 
   mercurial = callPackage ../applications/version-management/mercurial {
-    inherit (pythonPackages) curses docutils hg-git dulwich;
```

Comment:
Omitting `hg-git` and `dulwich` here is unnecessary. These dependencies will not be built if `gitSupport` isn't enabled.